### PR TITLE
Updates to Read S2 AOM JSON Files

### PR DESCRIPTION
**Pull request by David-N-Perkins**
These are the changes required to read S2 AOM JSON files and for publishing our own Graphite version of Archie.

### CODE REVIEW

File: aom/src/main/java/com/nedap/archie/rminfo/ArchieModelNamingStrategy.java [hunk 1]
```diff
@@ -27,12 +27,12 @@ public ArchieModelNamingStrategy(boolean standardCompliantExpressionNames) {
...
+    private static final HashMap<String, String> rulesArchieToStandardTypeNamesMap = new HashMap<>();
+    static {
         rulesArchieToStandardTypeNamesMap.put("Operator", "EXPR_OPERATOR");
         rulesArchieToStandardTypeNamesMap.put("UnaryOperator", "EXPR_UNARY_OPERATOR");
         rulesArchieToStandardTypeNamesMap.put("BinaryOperator", "EXPR_BINARY_OPERATOR");
-        rulesArchieToStandardTypeNamesMap.put("Leaf", "EXPR_LITERAL");
+        rulesArchieToStandardTypeNamesMap.put("Leaf", "EXPR_LEAF");
         rulesArchieToStandardTypeNamesMap.put("Function", "EXPR_FUNCTION");
         rulesArchieToStandardTypeNamesMap.put("VariableReference", "EXPR_VARIABLE_REF");
         rulesArchieToStandardTypeNamesMap.put("Constant", "EXPR_CONSTANT");
```
Comment (wolandscat) [inline aom/src/main/java/com/nedap/archie/rminfo/ArchieModelNamingStrategy.java:38]: The line `rulesArchieToStandardTypeNamesMap.put("Constant", "EXPR_CONSTANT");` should be `  rulesArchieToStandardTypeNamesMap.put("Constant", "EXPR_LITERAL");`

File: aom/src/main/java/com/nedap/archie/rules/ExpressionType.java [hunk 2]
```diff
@@ -47,6 +49,8 @@ public String toString() {
                 return "DateTime";
             case DURATION:
                 return "Duration";
+            case C_STRING:
+                return "C_STRING";
```
Comment (wolandscat) [inline aom/src/main/java/com/nedap/archie/rules/ExpressionType.java:53]: I would guess this should be the name of the type in CamelCase, i.e. `return "CString";`

**Review by wolandscat — CHANGES_REQUESTED** [review]

**Review by borutjures — APPROVED** [review]
Looks good to me.

File: aom/src/main/java/com/nedap/archie/rminfo/ArchieModelNamingStrategy.java (hunk 1 above)
Comment (David-N-Perkins) [reply aom/src/main/java/com/nedap/archie/rminfo/ArchieModelNamingStrategy.java]: Done.

File: aom/src/main/java/com/nedap/archie/rules/ExpressionType.java (hunk 2 above)
Comment (David-N-Perkins) [reply aom/src/main/java/com/nedap/archie/rules/ExpressionType.java]: Fixed.

**Review by wolandscat — APPROVED** [review]
All good from my POV

**Pull request merged**